.github/workflows/contracts.yml

```
@@ -0,0 +1,52 @@
+  pull_request:
+    branches: ['main']
+    paths:
+      - 'aqua_contract/**'
+      - '.github/workflows/contracts.yml'
+
+jobs:
+  contracts:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
```

actionlint (1.7.7)

the runner of "actions-rs/toolchain@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)